Source/WebCore/dom/TreeScope.h

```diff
@@ -28,6 +28,7 @@
 
 #include "ExceptionOr.h"
 #include "TreeScopeOrderedMap.h"
+#include <JavaScriptCore/JSCJSValue.h>
```

Comment:
We try so hard to keep unnecessary header includes out of headers, which over time can help us avoid slowing down build times. Instead of adding this include, I suggest we add a forward declaration:
```
namespace JSC {
class JSValue;
}
```
That should fix compilation without pulling another header into every file that uses `TreeScope.h`.

Reply:
Of course, typically we prefer to forward-declare types in the headers where possible. In this case `TreeScope.h` returns a plain `JSC::JSValue` so it seemeed like the compiler would need to know the size of the type... but I tried and interestingly enough the forward declaration works in this particular instance 👌🏼